Sources/ITwinMobile/ITMViewController.swift

```diff
@@ -18,6 +18,7 @@ open class ITMViewController: UIViewController {
     public private(set) var itmNativeUI: ITMNativeUI?
     private var loadedOnce = false
     private var willEnterForegroundObserver: Any? = nil
+    private static var activeVC: ITMViewController?
```

Comment:
What's the purpose of this property?

Reply:
In the ThirdPartyAuth sample, I added the ability to go back to the login screen. If you do that and then go back to the Model View, it used to crash with an unhandled exception due to the fact that a new ITMViewController was created by SwiftUI, and the model view was still attached to the old ITMViewController. If you look at my change below, you will see that I check if activeVC has a value, and if it does I switch that (old) ITMViewController to have a generic UIView as its view (thus detaching the model view). I then store the new ITMViewController in activeVC and attach the model view to it.

Reply:
We should probably add a comment in the loadView code that explains this.

Reply:
Documented.